README.md

```diff
@@ -542,6 +542,16 @@ amadeus.dutyOfCare.diseases.covid19Report.get({
   language: 'EN'
 });
 
+//Car & Transfers APIs
```

Comment:
Car-> [Cars](https://developers.amadeus.com/self-service/category/cars-and-transfers)